mrmustard/lab_dev/states/base.py

```diff
@@ -248,11 +253,34 @@ def fock_array(self, shape: Optional[Union[int, Sequence[int]]] = None) -> Compl
         """
         return to_fock(self.representation, shape).array
 
-    def phase_space(self) -> tuple[ComplexMatrix, ComplexVector]:
+    def phase_space(self, s: float, characteristic: bool) -> tuple[ComplexMatrix, ComplexVector]:
         r"""
-        The covariance matrix and the vector of means that describe this state in phase space.
+        The s-parametrized information about the state that describe this state in phase space.
```

Comment:
Can you leave a bit more comments about this?
I am thinking about a user (e.g., me) that sees this for the first time and reads that this gives "s-parametrized information." They'll certainly wonder what this information means.

Reply:
@ziofil Could you have a look at this doc? I'm struggling in explaining this a little bit more clear in a way....